### PR TITLE
Some comments added to English strings.xml

### PR DESCRIPTION
**Pull request by MatiasMFM2001**
**Describe the pull request**

I've added some comments so @corbinlc won't need to explain other translators what he has explained to me. Maybe you should add other comments!

**Link to relevant issues**
#500 #430 #494

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
-<resources>
+<resources>                                                         <!-- Translators, pay attention to this notes! -->
```
Comment (EnderNightLord-ChromeBook) [inline app/src/main/res/values/strings.xml]: pay attention to this notes

hows about 

pay attention to these notes

Comment (MatiasMFM2001) [reply app/src/main/res/values/strings.xml]: You're right! Thanks for marking that!

Comment (EnderNightLord-ChromeBook) [reply app/src/main/res/values/strings.xml]: your welcome

Comment (EnderNightLord-ChromeBook) [reply app/src/main/res/values/strings.xml]: And Everything Else Looks Good.

**Pull request merged**